### DISCUSSION TITLE
run VpnHood Server on Linux

### DISCUSSION THREAD

**Discussion by abolfazl6** [vpnhood/VpnHood#433] [Q&A]
Hello, recently when I install vpnhood on the server and I want to create an access token, I encounter this error:
  "Data": {},
  "TypeName": "InvalidOperationException",
  "TypeFullName": "System.InvalidOperationException",
  "Message": "Could not find any public access point for the ServerFarm. Please configure a server for this AccessToken."
}

Do you have a solution?

**Comment by trudyhood** [comment]
Hi,
Can you share me your project id?

**Comment by abolfazl6** [comment]
Hello, I searched and found that it should be in the server section first
I turned off Automatic Configuration and then it gave me the access key, but it doesn't work for the user and I faced other errors that I am sending to you.
Project ID: cce95c2b-61ee-4f86-a76a-7002b7b8f29b

**Comment by abolfazl6** [comment]
When I type the following command in the server

 sudo /opt/VpnHoodServer/vhserver start

It gives the text that you see in the picture
![Screenshot 2024-01-03 154700](https://github.com/vpnhood/VpnHood/assets/87604379/d28a1b02-9a82-4efd-8890-7f0040c9ea5c)

**Comment by trudyhood** [comment]
OK, now I need your server log file. /opt/VpnHoodServer/storage/logs/server.log
You can also share your server login credentials so I can check it out for you and make it work. trudy@vpnhood.com
